tests/test_anotate.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from unittest.mock import patch
+from sigpac_tools.anotate import get_metadata
+
+# Mock data for the test
+layer_parcela_response = {"key": "value"}
+layer_recinto_response = {"key": "value2"}
+
+
+class TestGetMetadata:
+    @patch("sigpac_tools.anotate.__query")
+    def test_parcela_layer(self, mock_query):
+        mock_query.return_value = layer_parcela_response
+        data = {
+            "province": 1,
+            "municipality": 1,
+            "polygon": 1,
+            "parcel": 1,
+        }
+
+        result = get_metadata("parcela", data)
+        assert result == layer_parcela_response
```

Comment:
This test is testing that result == result.

Bad tests are worse than no test, remove unnecesary tests and leave only ones that test useful features.

Reply:
Would it be nicer to remove those tests or to try to evaluate the resultant dictionary with a json.schema?

Reply:
I have finally removed unuseful tests

Reply:
> Would it be nicer to remove those tests or to try to evaluate the resultant dictionary with a json.schema?

As long as the test is not doing any work outside of calling another function/API(mocked or not) I do not believe this to be necesary